### PR TITLE
WIP: --rm flake

### PR DESCRIPTION
**Pull request by baude**
this is a test to see if we can illuminate why the test flakes
periodically.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: test/e2e/start_test.go [hunk 1]
```diff
@@ -108,7 +109,7 @@ var _ = Describe("Podman start", func() {
 		start := podmanTest.Podman([]string{"start", "-l"})
 		start.WaitWithDefaultTimeout()
 		Expect(start.ExitCode()).To(Not(Equal(0)))
-
+		time.Sleep(2 * time.Second)
```
Comment (rhatdan) [inline test/e2e/start_test.go]: Why not do a 
 		wait := podmanTest.Podman([]string{"wait", "-l"})

Comment (baude) [reply test/e2e/start_test.go]: we are just testing a theory here.

**Comment by rhatdan** [comment]
Any conclusions?

**Comment by vrothberg** [comment]
Just noticed this PR. I opened https://github.com/containers/libpod/issues/4167 earlier today.

**Pull request closed without merging**